contracts/gnosis/AnteGnosisPlungeTest.sol

```diff
@@ -0,0 +1,45 @@
+pragma solidity ^0.8.0;
+
+import "../AnteTest.sol";
+import "../interfaces/IERC20.sol";
+
+// @title Gnosis Plunge Test
+contract GnosisTVLPlungeTest is AnteTest("Make sure the TVL is at least 15% of the original TVL") {
+
+    IERC20 private constant USDT = IERC20(0xdAC17F958D2ee523a2206206994597C13D831ec7);
+    IERC20 private constant USDC = IERC20(0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48);
+    IERC20 private constant DAI = IERC20(0x6B175474E89094C44Da98b954EedeAC495271d0F);
+    IERC20 private constant UST = IERC20(0xa47c8bf37f92aBed4A126BDA807A7b7498661acD);
+    IERC20 private constant WETH = IERC20(0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2); 
+
+    IERC20 private constant GNOSIS_CONTRACT = IERC20(0x6F400810b62df8E13fded51bE75fF5393eaa841F);
```

Comment:
This variable isn't used at all, maybe you can make this just the address, so you can reuse it below, because I see it used many times below.

And if that's the case, let's clarify what this address is and how it relates to the gnosis protocol so that readers don't have to go digging around to look for it.